data/cartridge.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Cartridge",
+  "description": "Starknet Gaming Console. Play your favorite games.",
+  "short_description": "Starknet Gaming Console",
+  "tags": ["Games"],
+  "contracts": [],
+  "goerliContracts": [],
+  "audits": [],
+  "dotw": false,
+  "links": {
+    "website": "https://cartridge.gg/",
+    "careers": "",
+    "twitter": "https://twitter.com/cartridge_gg",
+    "telegram": "",
+    "discord": "https://discord.gg/cartridge",
+    "github": "https://github.com/cartridge_gg",
+    "youtube": "",
+    "medium": "",
+    "mirror": ""
+  },
+  "twitterName": "cartridge_gg",
+  "teamInfo": {
+    "founded": "2022-04-00T00:00:00.000Z",
+    "anonymous": false
+  },
+  "tokens": [],
+  "media": {
+    "logoUrl": "https://static.cartridge.gg/logo.jpeg",
+    "bannerUrl": "https://static.cartridge.gg/banner.png",
+    "previewUrl": "https://static.cartridge.gg/preview.jpeg",
```

Comment:
In order for this to work, you'll need to add the domain [in next.config.js](https://github.com/argentlabs/dappland/blob/develop/next.config.js) like so:

```
  images: {
    domains: [
      "dv3jj1unlp2jl.cloudfront.net",
      "pbs.twimg.com",
      "github.com",
      "eykar.org",
      "drive.google.com",
      "mintsquare.sfo3.cdn.digitaloceanspaces.com",
      "www.frenslands.xyz",
      "raw.githubusercontent.com",
      "lh3.googleusercontent.com",
      "static.cartridge.gg" --> ADD THIS LINE
    ],
  }, 
```